Rename "dissoluted devourer" to "dissolving devourer"

autarch opened the pull request:
#### Summary

None

#### Purpose of change

The word "dissoluted" has several problems. First, it's not a word, because "dissolute" is an adjective, not a verb. Second, "dissolute" means "debauched", not "dissolving" or "dissolved".

#### Describe the solution

I renamed this to "dissolving devourer".

#### Additional context

I think there are a few other names that could work:

* dissolved devourer
* melted devourer
* melting devourer

And more along those lines. I think "dissolving" is good, but I'm fine with of these choices.

Bone-blood-moss commented:
Dissoluted works better zombies Dissolved into it and all. it tells you what the and what's happening and what it does.

autarch commented:
> Dissoluted works better zombies Dissolved into it and all. it tells you what the and what's happening and what it does.

As I said, "dissoluted" is simply not a word in the English language. The word "dissolute" is an adjective, not a verb. You can add a "d" to make it past tense.

But it sounds like maybe you're saying you prefer _"dissolved"_ over "dissolving". I'm fine with either, but the reason I picked "dissolving" is that "dissolved" makes it sound like it's already a puddle. The word "dissolving" can suggest that it's either in the process of dissolving itself _or_ that it dissolves others, which I think is a bit better.

autarch committed:
Rename "dissoluted devourer" to "dissolving devourer"

The word "dissoluted" had several problems. First, it's not a word, because "dissolute" is an
adjective, not a verb. Second, "dissolute" means "debauched", not "dissolving" or "dissolved".

Vaskritaya commented:
To me, words like "melting" and 'dissolving" bring to mind acid, which is a pretty common archetype for zombies and doesn't reflect what the devourer actually is. "dissoluted" was functionally meaningless to me and probably most others, but I'm not sure this is an improvement in clarity.

autarch commented:
If folks like the word "dissolute" then I think the best option is to rename it to "dissolute devourer". I think it's a bit of a goofy name if you know what dissolute means, though. Is this fusion of zombies is drinking, gambling, and hanging around with people of low character in Victorian England or something?

oosyrag commented:
There is no need to maintain alliteration. 

Colossal devourer 
Zombie devourer 
Zombie amalgam/amalgamation
Melded zombie

I personally do not like dissolving (or dissolved) devourer at all. It implies it itself is dissolving or that it dissolves other things, neither of which are particularly suitable.

XygenSS commented:
"zombie amalgamation" is problematic because we already have flesh-raptors and other "amalgamation" zombie monsters. Also, we have an "anatomy of amalgamations" proficiency as well.

oosyrag commented:
> "zombie amalgamation" is problematic because we already have flesh-raptors and other "amalgamation" zombie monsters. Also, we have an "anatomy of amalgamations" proficiency as well.

The devourer predates the newer amalgamations, that doesn't mean it can't be updated. Is there any other particular reason it being an amalgamation is a problem?

XygenSS commented:
All "amalgamations" are special classes of monsters, notably ones spawned by the Shadow, https://cdda-guide.nornagon.net/search/amalgamation

an ordinary zombie having an overlap is undesirable IMO because it implies they belong in the same category. However one is human in origin, the other is significantly more alien

autarch commented:
I'm really not too hung up on any particular name. It just drives me nuts every time I see "dissoluted". So anything that uses actual words is fine with me!

GuardianDll commented:
i too agree better name would be nice, and i too agree `dissolving` is misleading here; so does amalgam, even if it fits the theme. what about "fused devourer"?

KittyTac commented:
Some ideas:
- mass of zombies
- writhing pile
- shambling devourer

XygenSS commented:
+1 to shambling devourer, melded devourer, zombie devourer, etc...

oosyrag commented:
> All "amalgamations" are special classes of monsters, notably ones spawned by the Shadow, https://cdda-guide.nornagon.net/search/amalgamation
> 
> an ordinary zombie having an overlap is undesirable IMO because it implies they belong in the same category. However one is human in origin, the other is significantly more alien

I'm well aware of that - the point was that there's no particular barrier to keep this monster as an ordinary zombie to begin with, as it can always be changed/updated. Anyways there's always room to make a new amalgamation that eats things to grow down the line anyway.

KheaHyena commented:
Didn't even know it wasn't a word, though I had an idea of it's meaning.
I just imagined that, in Zombie society, it'd be extremely rude to grab one of yours and shove them into your body mass and create a bigger agglomerate (which is what I kinda pictured, unrestrained in grabbing nearby zombies just to devour them, in a dissolute frenzy, as far zombies go at least).
_Do wish they were a bigger threat though, or at least become one, but I digress._

If name change\correction is truly desired, I do agree that **backing away** from anything related to acid (i.e.: melting, dissolving, etc...) is more appropriate and clarifying to anyone who first meets this enemy.

I do like the word **Frenzied Devourer**, but even the simple **Shambling** or even **Zombie Devourer** work great (not really the **Melded**, we already have the **Melded Task Force** for that).
And if a type of alliteration is wanted, one could always try... **Delirious** or **Deriding** | **Devourer**.
